public/app/features/dashboard/dashgrid/PanelHeader/PanelHeader.tsx

```diff
@@ -28,7 +28,18 @@ export interface Props {
 
 export const PanelHeader: FC<Props> = ({ panel, error, isViewing, isEditing, data, alertState, dashboard }) => {
   const onCancelQuery = () => panel.getQueryRunner().cancelQuery();
-  const title = panel.getDisplayTitle();
+  const statsExtendVar = {
+    queries_number: data.series.length,
```

Comment:
This is not true , the series length has nothing to do with number of queries. A single query can result in 0-N series